rules/manager.go

```diff
@@ -37,9 +37,9 @@ import (
 const (
 	namespace = "prometheus"
 
-	ruleTypeLabel     = "rule_type"
-	alertingRuleType  = "alerting"
-	recordingRuleType = "recording"
+	RuleTypeLabel     = "rule_type"
```

Comment:
Well, as mentioned the package was very inconsistent. `alerting.go` was exporting all its label/value constants such as `AlertLabelName` so I made the ones in `manager.go` consistent with those.
Not caring too much in this place, but then the other constants have to become unexported.

In general though, being strict about this, both rules would not have public constructors, the `Rule` interface would not have an `Eval` but an `eval` method. The `Rule` interface would not have a `String()` method. `AlertingRule` would be exported but `RecordingRule` not.